### PR TITLE
- task 100

### PR DESCRIPTION
**Pull request by artsiomMandrukevich**
added Allure Reporting

### CODE REVIEW

File: src/main/java/pages/LogInPage.java [hunk 1]
```diff
@@ -25,11 +28,17 @@ public void clickSignInButton() {
...
         enterUserName(userName);
         clickSignInButton();
         enterPassword(password);
         clickSignInButton();
         return new PersonalAccountPage(driver);
     }
+
+    @Step("Check the log out happened correctly (comparing titles)")
+    public void assertTitleAfterLogOut(String actTitle, String expTitle) {
+        assertEquals(actTitle, expTitle);
```
Comment (LikBez123) [inline src/main/java/pages/LogInPage.java]: I don't think you need to have this method with assertions here at the page layer

File: src/main/java/pages/PersonalAccountPage.java [hunk 2]
```diff
@@ -25,6 +28,12 @@ public void clickLogOut() {
         driver.findElement(LOG_OUT_BUTTON).click();
     }
 
+    @Step("Check actualUserAccountName =  expectedUserAccountName")
+    public void assertUserAccountName(String actName, String expName) {
+        assertEquals(actName, expName);
```
Comment (LikBez123) [inline src/main/java/pages/PersonalAccountPage.java]: the same

**Review by LikBez123 — APPROVED** [review]

File: src/main/java/pages/LogInPage.java (hunk 1 above)
Comment (artsiomMandrukevich) [reply src/main/java/pages/LogInPage.java]: fixed

File: src/main/java/pages/PersonalAccountPage.java (hunk 2 above)
Comment (artsiomMandrukevich) [reply src/main/java/pages/PersonalAccountPage.java]: fixed

**Pull request merged**